src/common/logging/log.h

```diff
@@ -6,11 +6,26 @@
 
 #include <algorithm>
 #include <string_view>
+#include <type_traits>
 
-#include <fmt/core.h>
+#include <fmt/format.h>
 
 #include "common/logging/types.h"
 
+// adapted from https://github.com/fmtlib/fmt/issues/2704
+// a generic formatter for enum classes (<= 32 bits)
```

Comment:
```suggestion
// a generic formatter for enum classes (<= 32 bits)
```
Since it now uses std::underlying_type_t, this comment no longer applies